app/retail/views.py

```diff
@@ -1030,6 +1029,12 @@ def create_status_update(request):
             response['message'] = 'Bad Request'
             logger.error('Status Update error - Error: (%s) - Handle: (%s)', e, profile.handle if profile else '')
             return JsonResponse(response, status=400)
+    else:
+        response['status'] = 401
+        response['message'] = 'Not logged in!'
+        print('here', response)
```

Comment:
we can take out the print statement

Reply:
@octavioamu can you remove this? Other than that I think this is good to go.